src/careamics/lightning/dataset_ng/callbacks/prediction_writer/write_tiles_zarr_strategy.py

```diff
@@ -341,7 +341,7 @@ def write_tile(self, dirpath: Path, region: ImageRegionData) -> None:
             # region.data has shape C(Z)YX, broadcast can fail with singleton dims
             crop = region.data[crop_slices]
 
-            if region.data.shape[0] == 1:
+            if region.data.shape[0] == 1 and "C" not in region.axes:
```

Comment:
Just a comment here: the choice of not writing to zarr singleton dimensions that do not exists in `axes` remain, but will be updated by writing to original shape. Now, in case where selecting a single channel via the `channels` parameter, channel dim can be a singleton but `C` be in `axes`. 

I chose to keep the axis here for now.